presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestAggregations.java

```diff
@@ -78,6 +78,20 @@ public void testCountWithNullIfPredicate()
         assertQuery("SELECT COUNT(*) FROM orders WHERE NULLIF(orderstatus, 'F') = orderstatus ");
     }
 
+    @Test
+    public void testAggregationPushdownThroughOuterJoinNotFiringInCorrelatedAggregatesLeftSide()
+    {
+        assertQuery("SELECT max(x) from (SELECT * from (VALUES 1) t(x) LEFT JOIN (VALUES 1) t2(y) ON t.x = t2.y) GROUP BY x",
+                "VALUES 1");
```

Comment:
@kokosing The test cannot use multiple values here because ``PushAggregationThroughOuterJoin`` checks for distinct output from the Join's Outer Table, which translates to ``ValuesNode`` having a single row. For multiple values, the rule will not run and the functionality will not be tested

Reply:
Sure. Thanks for checking.